### PR TITLE
Tickets/sitcom 537

### PR DESCRIPTION
**Pull request by isotuela**
Hi Patrick, 
I have added the check bestEffortISR.get_exposure() for the three filters, and also moved prepare_for_flatfield at the end of the notebook to make a smoother transition to calibrations.

### CODE REVIEW

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb [hunk 1]
```diff
@@ -22,13 +22,17 @@
     "import time\n",
```
Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>why don't you want to write? </p><p>I think it might be worthwhile in order to verify permissions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <pre class="ql-syntax" spellcheck="false">Line #6.        time.sleep(10)
</pre><p>So first any sleeps should use <code>await asyncio.sleep(time_in_seconds)</code></p><p>However, see the example above for the bias? I think you want to catch the ingestion event instead of sleep </p><p><code>&nbsp;&nbsp;ingest_event = await latiss.rem.atoods.evt_imageInOODS.next(flush=False, timeout=10)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Why did you move this down?</p><p> </p><p>It was up top because it helps diagnose issues... in fact does this even work? I don't think you've enabled anything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='68'/>

**Comment by patrickingraham** [comment]
I know you're going to add the filter checks during slews but thought I'd take a look at this first anyways.

**Comment by patrickingraham** [comment]
Sorry, didn't mean to close that!

File: procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb (hunk 1 above)
Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Yes, thanks for the clarification!</p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>What do you think If I move back the checks where they were, and leave the section "Run prepare for flats" at the end of the notebook. This way calibrations can start immediately after that. </p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Great idea. Thanks!</p><p> </p>

Comment (patrickingraham) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>I guess I'm not understanding why you want to change it. Is this to save time?</p><p>The reason prepare for flats runs first is because it tests out the mirror covers and pneumatics etc. </p><p>It also moves in and out of the dome following.</p><p>In the future this will be a script, or more likely a series of scripts, might be best to just leave it.</p>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Please add a comment somewhere that tells people to make sure they check rubinTV to make sure the images are appearing.</p><p>That's the canary in the coal mine :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (patrickingraham) [inline procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>You call this same block of code 4 times.</p><p>Best to make it a function.</p><p>I'd suggest using target 1 to cycle through all filters, with grating=empty,</p><p>then target 2 can be all filters with grating = hologram.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Now it's only called twice so I have left it as it was, so users can see also the code easier. </p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>I have implemented images in all instrument setups. Thanks!</p><p> </p>

Comment (isotuela) [reply procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb:22]: <p>Done</p>

**Comment by patrickingraham** [comment]
Looks good. Thanks!
Can you rebase then squash commits? Then we'll merge this!

**Review by patrickingraham — APPROVED** [review]
Thanks for making these changes!
